packages/api-derive/src/dappsStaking/index.ts

```diff
@@ -8,6 +8,9 @@ import { ContractAddress } from '../types';
 
 export const getAddressEnum = (address: string) => ({ Evm: address });
 
+/**
+ * @deprecated stakers does not return results
```

Comment:
is there a replacement function for this? Or is this no longer needed completely?

Reply:
This doesn't work currently as well. Dapp staking pallet has stopped sending stakers.